src/system/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { StoryObj } from '@storybook/react';
+
+import { Footer } from '../Footer/Footer';
+
+export default {
+	title: 'Footer',
```

Comment:
What do you think about having `Navigation/Footer`? Since Footer is part of the navigation components, I believe it makes sense.

Reply:
I agree!